### PR TITLE
Fix graph-editor css

### LINKED ISSUE #169: Fix css in graph-editor
This is for either @oveddan or @beeglebug, who ever can fix it.  The graph editor currently looks like this:

Screenshot 2022-11-29 at 8 01 54 PM

### PR DESCRIPTION
**Pull request by oveddan**
Have the example app use tailwind to generate styles for the imported flow package, thus enabling tailwind to be applied to the imported components. This fixes #169 .

This solution works but has a couple shortcomings:

In the example config, we now have it setup like:
```js
module.exports = {
  content: [
    './index.html',
    './src/**/*.{js,ts,jsx,tsx,css}',
    '../../packages/react-flow/dist/**/*.{js,ts,jsx,tsx,css}'
  ],
  theme: {
    extend: {}
  },
  plugins: []
};
```

The fix adds the last time under `content` which instructs tailwind to generate the necessary css styles for the files in the flow lib, but it has to find them by relative path.

If someone were to build their own flow editor that imported these components, they would have to configure it have that path come from `node_modules`:
```js
module.exports = {
  content: [
    './index.html',
    './src/**/*.{js,ts,jsx,tsx,css}',
    'node-modules/@behave-graph/react-flow/dist/**/*.{js,ts,jsx,tsx,css}'
  ],
  theme: {
    extend: {}
  },
  plugins: []
};
```

Second shortcoming is that for some reason, if you run `preconstruct dev` from the root, tailwind doesnt get applied to imported components from the `flow` lib, but if you run `preconstruct build` it does

My thoughts is this works for now, and in the future we can come up with a better way, maybe we can use [flowbite-react](https://github.com/themesberg/flowbite-react) as an example.

### CODE REVIEW

File: examples/graph-editor/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>@Behave-Graph/React-Flow Graph Editor Example</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (oveddan) [inline examples/graph-editor/index.html:5]: Nice to have improvement not quite related to the PR

File: packages/react-flow/tsconfig.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "extends": "../../tsconfig.json",
   "compilerOptions": {
-    "jsx": "react",
+    "jsx": "react-jsx",
```
Comment (oveddan) [inline packages/react-flow/tsconfig.json:4]: unrelated to PR but should get rid of the need to `import "react"`

**Comment by oveddan** [comment]
Just fixed the merge conflicts from the recent PR merge #171

**Comment by bhouston** [comment]
Thank you!  I got it in!  This is so exciting and great collaboration!

**Pull request merged**